File: pfaProject/ASP.NET/PfaBackFinal/Models/Candidature.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace HdBackApp.Models
{
    public class Candidature
    {
        [Key]
        public int Id { get; set; }
        public Candidat Candidat { get; set; }
        public int CandidatId { get; set; }
        public string Offre { get; set; }
        public DateTime DateCandidature { get; set; }
        public   Historique historique { get; set; }

        public Candidature()
        {
            historique = new Historique();
        }
    }
}

File: pfaProject/Angular/src/app/models/Candidature.ts
import { historique } from './historique';
import { Candidat } from './Candidat';

export class Candidature {

   id:number;
    Candidat:Candidat;
    CandidatId:number;
    offre:string;
    dateCandidature:Date;
    historique:historique;
}
